resources/views/articles/show.blade.php

```diff
@@ -1,7 +1,7 @@
 @extends('layouts.default')
 
 @push('meta')
-<link rel="canonical" href="{{ $article->originalUrl() }}" />
+<link rel="canonical" href="{{ $article->canonicalUrl() }}" />
```

Comment:
Is this needed? We're not showing the article underneath a different url.

Reply:
The canonical URL will be either the one that is provided by the user pointing at a the original location or it will be the Laravel.io URL.

To me, these are two different things.

So `originalUrl()`returns the value entered by the user and `canonicalUrl()` returns that if it exists or the Laravel.io URL if not.

In my opinion we need both. Even thinking about the edit page, you wouldn’t want the `original_url` field populates bu the Laravel.io URL.

Hope that all makes sense?

Reply:
I see. The current implementation seems okay then 👍